frontend/playground-frontend/src/components/Dashboard/Dashboard.js

```diff
@@ -71,16 +77,16 @@ const StatusDisplay = ({ statusType, status }) => {
         Queued: {status}
       </button>
     );
-  } else if (statusType === "training") {
+  } else if (statusType === "STARTING") {
```

Comment:
remember, there are more statuses. See [execution_db.py](https://github.com/DSGT-DLP/Deep-Learning-Playground/blob/dev/backend/aws_helpers/dynamo_db_utils/execution_db.py): 24

Reply:
The only statuses that I saw being returned from was dynamodb though was starting and success (although that could be because I am running dlp in dev mode)

Reply:
there are more statuses @dwu359 . faris is right on this